CHANGELOG.md

```diff
@@ -1,5 +1,4 @@
 ## 2.1.4
```

Comment:
Add an entry here about Dart SDK >2.0.0...

Reply:
I just updated the CHANGELOG.md since version 2.1.4 hasn't been released yet, so 2.1.4 is the version bump.